packages/sanity/src/_singletons/index.ts

```diff
@@ -22,6 +22,7 @@ export * from './core/form/studio/PresenceContext'
 export * from './core/form/studio/ReferenceInputOptionsContext'
 export * from './core/form/studio/reviewChanges/ReviewChangesContext'
 export * from './core/form/studio/ValidationContext'
+export * from './core/form/tree-editing'
```

Comment:
Forgive me for the possible dumb question, but I'm not sure what you mean by typos in this instance.
In this case there is an index file on the tree-editing folder. I understand that's not how we want to do things forward, but I might be missing something unrelated to the barrel exports

Reply:
oh sorry I meant since I typed the suggestion wanted to make sure I did not have any typos in my suggestion I commented. Anywho your changes are correct ignore me :homer-disappear: